### PR TITLE
Redesigning `protostar.toml` 13 — Rolling integration — Package management commands

### PR DESCRIPTION
**Pull request by kasperski95**
Add `lib-path` argument to package management commands. This argument is used after migrating to the `ConfigurationFileV2`. Make lib directory optional.

Related #477

### CODE REVIEW

File: protostar/cli/common_arguments.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from .protostar_argument import ProtostarArgument
+
+lib_path_arg = ProtostarArgument(
+    name="lib-path",
+    description="Directory containing project dependencies. "
+    "This argument is/will after migrating the configuration file.",
```
Comment (Arcticae) [inline protostar/cli/common_arguments.py:6]: This seems a bit wrong

File: protostar/cli/common_arguments.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from .protostar_argument import ProtostarArgument
+
+lib_path_arg = ProtostarArgument(
```
Comment (Arcticae) [inline protostar/cli/common_arguments.py:3]: Shouldn't this have a default of value "lib"?

Comment (Arcticae) [inline protostar/cli/common_arguments.py:3]: Constants should be all uppercase

File: tests/integration/protostar_fixture.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import asyncio
```
Comment (Arcticae) [inline tests/integration/protostar_fixture.py:1]: Please revert formatting-only changes

**Review by Arcticae — CHANGES_REQUESTED** [review]
I understand that removing this argument from config file is technically simplifying the `.toml`, but this instead requires passing this on every single invocation of a command. I think this is totally unnecessary and will result in a worse UX.

File: tests/integration/protostar_fixture.py (hunk 3 above)
Comment (kasperski95) [reply tests/integration/protostar_fixture.py:1]: [revert formatting](https://github.com/software-mansion/protostar/pull/1058/commits/0fb106927da80f05ce082d01368ed980a4acd741)

File: protostar/cli/common_arguments.py (hunk 2 above)
Comment (kasperski95) [reply protostar/cli/common_arguments.py]: [use constant case](https://github.com/software-mansion/protostar/pull/1058/commits/009c1322db2dabc6584dd819891e0ddbe131be08)

Comment (kasperski95) [reply protostar/cli/common_arguments.py]: We can't use default if the ConfigurationFileV1 is used. Protostar needs to print the warning if the user decides to pass the argument explicitly and the Configuration File isn't migrated. LibPathResolver handles the default value logic for the time being.

File: protostar/cli/common_arguments.py (hunk 1 above)
Comment (kasperski95) [reply protostar/cli/common_arguments.py]: [update description](https://github.com/software-mansion/protostar/pull/1058/commits/9e7079d021f1ed070be5939059f2b83f78125d86)

**Review by Arcticae — APPROVED** [review]

**Pull request merged**